lib/teselagen/api/tests/test_test_client.py

```diff
@@ -52,27 +52,33 @@ def client(self, host_url: str, api_token_name: str) -> TeselaGenClient:
 
         """
         test_client = TeselaGenClient(api_token_name=api_token_name,
-                                      host_url=host_url)
+                                      host_url=host_url,
+                                      module_name='test')
         return test_client
 
     @pytest.fixture
-    def logged_client(self, client: TeselaGenClient,
-                      expiration_time: str) -> TESTClient:
+    def logged_client(
```

Comment:
Could we transform the `logged_client` fixtures (which are defined in this and other test files) into `yield fixtures`? - similarly to what was done with the `experiment` fixture (possibly using [`autouse = True`](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request)), so that they are logged in before starting the execution of each test and are automatically logged out when they finish.